src/libtego_ui/ui/qml/main.qml

```diff
@@ -120,18 +120,18 @@ QtObject {
             property int dialogWindowFlags: Qt.Dialog | Qt.WindowSystemMenuHint | Qt.WindowTitleHint | Qt.WindowCloseButtonHint
         },
 
-        Timer {
-            interval: 2000
-            running: true
-            repeat: false
-            onTriggered: {
-                var pendingRequests = userIdentity.contacts.incomingRequests.requests
-                for (var i = 0; i < pendingRequests.length; i++) {
-                    var object = createDialog("ContactRequestDialog.qml", { 'request': pendingRequests[i] })
-                    object.visible = true
-                }
-            }
-        },
+        // Timer {
+        //     interval: 2000
+        //     running: true
+        //     repeat: false
+        //     onTriggered: {
+        //         var pendingRequests = userIdentity.requests
+        //         for (var i = 0; i < pendingRequests.length; i++) {
+        //             var object = createDialog("ContactRequestDialog.qml", { 'request': pendingRequests[i] })
+        //             object.visible = true
+        //         }
+        //     }
+        // },
```

Comment:
why remove this?

Reply:
That timer was originally a bit of hack to avoid a race condition when loading up contacts from the json. The new API will properly fire a callback which triggers the requestAdded signal when we start the onion service with a list of pending incoming contact requests.